README.md

```diff
@@ -728,7 +728,7 @@ In either case:
       end
     ```
 
-* <a name="unless-with-comparator"></a>Avoid `unless` with comparators if you can use `if` with an opposing comparator.<sup>[[link](#unless-with-comparator)]</sup>
+* <a name="unless-with-comparison-operator"></a>Avoid `unless` with comparison operators if you can use `if` with an opposing comparison operator.<sup>[[link](#unless-with-comparison-operator)]</sup>
```

Comment:
comparison operator is more googlable (searching for ruby comparator gives comparison operator in the titles) so i think it's worth changing to avoid having to explain it here